### PR TITLE
fix: update readme

### PR DESCRIPTION
**Pull request by stevanlohja**
Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,95 @@
...
-</p>
-
-<center>
+<!-- project shields -->
   <span>
     <img alt="CircleCI branch" src="https://img.shields.io/circleci/project/github/etclabscore/jade-service-runner/master.svg">
     <img src="https://codecov.io/gh/etclabscore/jade-service-runner/branch/master/graph/badge.svg" />
     <img alt="Dependabot status" src="https://api.dependabot.com/badges/status?host=github&repo=etclabscore/jade-service-runner" />
-    <img alt="Chat on Discord" src="https://img.shields.io/badge/chat-on%20discord-7289da.svg" />
+    <img alt="Chat on Discord" src="https://img.shields.io/badge/chat-on%20discord-7289da.svg?link=https://discord.gg/S9AT3X2" />
```
Comment (shanejonas) [inline README.md]: this should be wrapped in an `a` tag right? not working as link in the preview for me

Comment (stevanlohja) [reply README.md]: I've updated the discord badge. Works now.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,95 @@
...
+  <h3 align="center">Jade Service Runner</h3>
+
+  <p align="center">
+    An awesome JSON-RPC service manager!
+    <br />
+    <a href="https://www.youtube.com/watch?v=Y-Wdg1hgMls&t=2660s">View Demo</a>
+    ·
+    <a href="https://github.com/etclabscore/jade-service-runner/issues/new/choose">Report Bug</a>
+    ·
+    <a href="https://github.com/etclabscore/jade-service-runner/issues/new/choose">Request Feature</a>
```
Comment (shanejonas) [inline README.md]:
```suggestion
    <a href="https://github.com/etclabscore/jade-service-runner/issues/new?assignees=&labels=&template=feature_request.md&title=">Request Feature</a>
```

File: README.md [hunk 3]
```diff
@@ -1,46 +1,95 @@
...
+</p>
+<center>
+  <h3 align="center">Jade Service Runner</h3>
+
+  <p align="center">
+    An awesome JSON-RPC service manager!
+    <br />
+    <a href="https://www.youtube.com/watch?v=Y-Wdg1hgMls&t=2660s">View Demo</a>
+    ·
+    <a href="https://github.com/etclabscore/jade-service-runner/issues/new/choose">Report Bug</a>
```
Comment (shanejonas) [inline README.md]:
```suggestion
    <a href="https://github.com/etclabscore/jade-service-runner/issues/new?assignees=&labels=&template=bug_report.md&title=">Report Bug</a>
```

**Pull request merged**